deliver/lib/deliver/options.rb

```diff
@@ -186,12 +186,12 @@ def self.available_options
         # release
         FastlaneCore::ConfigItem.new(key: :automatic_release,
                                      env_name: "DELIVER_AUTOMATIC_RELEASE",
-                                     description: "Should the app be automatically released once it's approved? (Can not be used together with `auto_release_date`)",
+                                     description: "Should the app be automatically released once it's approved? (Cannot be used together with `auto_release_date`)",
```

Comment:
`Cannot`/`cannot` should generally be written without spaces except in exceedingly rare circumstances (which IMO should be avoided by rewording, but don't apply here)